cmd/agent/daemon/app/app.go

```diff
@@ -165,8 +167,15 @@ func (a *App) Run(ctx context.Context) error {
 			log = logging.New(logCfg)
 		}
 		exporters = state.NewExporters(log)
-		exporters.Events = append(exporters.Events, state.NewCastaiEventsExporter(log, castaiClient, a.cfg.ExportersQueueSize))
-		exporters.ContainerStats = append(exporters.ContainerStats, state.NewCastaiContainerStatsExporter(log, castaiClient, a.cfg.ExportersQueueSize))
+		if cfg.EBPFEventsEnabled {
```

Comment:
`nit`: `EBPFEvents` has more configs so perhaps could be extracted into struct, like `cfg.Network.Enabled`?